### PR TITLE
Add macos, linux, windows teams

### PR DESCRIPTION
**Pull request by cbracken**
The team-desktop label is being replaced by the team-linux, team-macos, team-windows labels in triage. This provides a more granular breakdown for issues and makes it easier for a combined iOS/macOS team to triage both, without also needing to wade through Windows and Linux issues.

As per instructions:
https://github.com/flutter/flutter/blob/master/docs/triage/README.md#adding-a-new-team

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [X] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [X] I have no idea who the current maintainers of Cocoon are.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: triage_bot/lib/engine.dart
Comment (stuartmorgan) [inline triage_bot/lib/engine.dart:64]: Shouldn't `desktop` (line 55) be removed as part of this?

Comment (cbracken) [reply triage_bot/lib/engine.dart:64]: I was assuming I'd remove desktop as a followup once the issue re-triage has concluded.

**Review by jmagman — APPROVED** [review]

**Comment by Hixie** [comment]
We should remove teams when we stop doing the triage for them, and add them when we start doing the triage, because this list really only controls what labels are checked for in the self-test issue that gets generated periodically. The rest of the bot is actually agnostic about the specific labels, it just looks for the prefixes.

**Comment by cbracken** [comment]
Sounds good -- done!

File: triage_bot/lib/engine.dart
Comment (cbracken) [reply triage_bot/lib/engine.dart:63]: Done.

**Comment by Hixie** [comment]
(I'll update the bot when I see this land, but ping me on Discord if I don't comment here that I've done it.)

**Comment by cbracken** [comment]
Thanks! I think I need an approval from you @Hixie since you're the only one listed in CODEOWNERS for this directory.

**Pull request merged**
